src/lib/platform/EiScreen.h

```diff
@@ -107,6 +109,19 @@ class EiScreen : public PlatformScreen {
     void on_motion_event(ei_event *event);
     void on_abs_motion_event(ei_event *event);
 
+    void handle_ei_log_event(struct ei *ei,
+                             enum ei_log_priority priority,
+                             const char *message,
+                             struct ei_log_context *context);
+
+    static void cb_handle_ei_log_event(struct ei *ei,
```

Comment:
I think this could be in the cpp file. This way we won't need exposing libei.h in the header.